### PR TITLE
Man30 refactor UI to class

### PR DESCRIPTION
**Pull request by tjohn11**

### CODE REVIEW

File: src/inventory/inventory.py [hunk 1]
```diff
@@ -92,20 +88,17 @@ def removeCardFromInventory(self, card, variant):
...
-        if current_count != None or 0:
-            query = f"UPDATE '{self.table_name}' SET {variant} = {current_count - 1} WHERE id == '{card['id']}';"
-            self.cursor.execute(query)
-            self.connection.commit()
-            return True
-        
-        return False
+        total = card[f'{variant}']
+        if total == None:
+            print('\n\nERROR: Cannot add Cards to Inventory.')
```
Comment (StevonShaw) [inline src/inventory/inventory.py]: Should be remove

File: src/ui/menu.py [hunk 2]
```diff
@@ -1,309 +1,231 @@
...
+            if selected_cards:
+                if action == 'add':
+                    self.__addCards(inventory, selected_cards)
+                elif action == 'remove':
+                    self.__removeCards(inventory, selected_cards)
+        except Exception as e:
+            print(e)
+            placeholder = input('')
+            
+    def __searchName(self, params):
```
Comment (StevonShaw) [inline src/ui/menu.py:163]: For `__addCards`, this `__searchName` function will search the whole listing of all 60k possible cards. --that is good we could add any one of them theoretically.

For `__removeCards`, it searches the same listing of 60k cards, and prompts the user to select one of those, even if the card is not actually in stock in the users database/inventory.

Can we make it so the `__removeCards` only passes a lists of cards that have a foil/nonfoil > 0?

**Review by StevonShaw — CHANGES_REQUESTED** [review]
Small typo, and if we can improve the remove function

File: src/ui/menu.py (hunk 2 above)
Comment (tjohn11) [reply src/ui/menu.py:163]: Will be implemented in another branch.

**Review by StevonShaw — APPROVED** [review]

**Pull request merged**